### DISCUSSION TITLE
8.2.0-beta2

### DISCUSSION THREAD

**Discussion by ferdicus** [rnmapbox/maps#1324] [General]
### Changes
* Add types for `Logger` class ([#1316](https://github.com/react-native-mapbox-gl/maps/pull/1316))  
* Enable linear easing on map camera ([#1281](https://github.com/react-native-mapbox-gl/maps/pull/1281))  
* Allow MapLibre as an option ([#1311](https://github.com/react-native-mapbox-gl/maps/pull/1311))  
* Fix native UserLocation on Android ([#1284](https://github.com/react-native-mapbox-gl/maps/pull/1284))   
* Add getClusterExpansionZoom to ShapeSource ([#1279](https://github.com/react-native-mapbox-gl/maps/pull/1279))  
* Add type definition for AnimatedPoint ([#1280](https://github.com/react-native-mapbox-gl/maps/pull/1280))  

<hr /><em>This discussion was created from the release <a href='https://github.com/react-native-mapbox-gl/maps/releases/tag/8.2.0-beta2'>8.2.0-beta2</a>.</em>

**Comment by ansarikhurshid786** [comment]
is It require to do following in latest version: 


Use pre_install hook to support non use_frameworks! usage #1262. Please add the following to your Podfile:
   pre_install do |installer|
     $RNMBGL.pre_install(installer)
     ...
   end
and
   post_install do |installer|
     $RNMBGL.post_install(installer)
     ...
   end

**Reply by mfazekas** [reply]
Yep it's still required if you don't use `use_frameworks!`
